devicon.json

```
@@ -1754,6 +1754,25 @@
         "color": "#ffcfab",
         "aliases": []
     },
+    {
+        "name": "gentoo", 
+        "tags": [
+            "linux",
+            "distribuition",
+            "desktop"
+        ], 
+        "versions": {
+            "svg": [
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain-wordmark"
```

```suggestion
                "plain-wordmark",
                "plain"
```
I think we can have the plain version too